api-lib/event_triggers/insertOrgMember.ts

```diff
@@ -0,0 +1,77 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import {
+  org_members_constraint,
+  org_members_update_column,
+} from '../gql/__generated__/zeus';
+import { adminClient } from '../gql/adminClient';
+import { EventTriggerPayload } from '../types';
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  try {
+    const payload: EventTriggerPayload<'users', 'INSERT' | 'UPDATE'> = req.body;
+    const user = payload.event.data.new;
+    const query = await adminClient.query(
```

Comment:
what you're trying to do here needs only a single query, because all the relationships are in the schema already (pseudocode below):
```
users_by_pk(id: userId)
  profile
    org_members(where: organization -> circle -> user -> id == userId)
```